test/base_helper.rb

```diff
@@ -29,6 +29,8 @@
 
 module TestApp
   class Application < Rails::Application
+    config.try :load_defaults, "#{Rails::VERSION::MAJOR}.#{Rails::VERSION::MINOR}"
```

Comment:
@AquisTech this one was important to set that configuration variable according to Rails version. But during the tests, it still doesn't generate the id, so I'm keep investigating